Lib/calendar.py

```diff
@@ -193,7 +193,7 @@ class Calendar(object):
     """
 
     def __init__(self, firstweekday=0):
-        self.firstweekday = firstweekday # 0 = Monday, 6 = Sunday
+        self.firstweekday = firstweekday
```

Comment:
Why is it removed?

Reply:
It seemed redundant.  I'll put it back if you wish.

Reply:
It is unrelated to this PR.

Reply:
I restored the comment.